spec/system/visit_change_password_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'ChangePassword', type: :system do
+  before(:each) do
+    login_gds_user
+  end
+
+  it 'shows form' do
+    visit profile_change_password_path()
+    expect(page).to have_content 'Old password'
+  end
+
+  it 'shows errors when an empty form is submitted' do
+    visit profile_change_password_path()
+    click_button 'Change password'
+    forms = page.find("#new_form > fieldset")
+    expect(forms).to have_content "Old password can't be blank"
```

Comment:
Can these be localised?

Reply:
I'm not sure. The errors don't utilise yml as far I can tell. I've tried looking at how the error messages are generated but i'm not really sure how it works 😕